### PR TITLE
Added subtraction method and tests.

### PR DESCRIPTION
**Pull request by evakaz**

### CODE REVIEW

File: src/Date.java [hunk 1]
```diff
@@ -79,46 +78,36 @@ public String displayDate() {
     }
     
     public void subscDays(int subDay) {
```
Comment (get-a-clue) [inline src/Date.java:80]: Let's rename to substractDays

File: src/Date.java [hunk 2]
```diff
@@ -79,46 +78,36 @@ public String displayDate() {
...
-    	int maxNumDays = getMaxNumDays(this.month, this.year);
+    	int maxNumDays = getMaxNumDays(this.month, this.year); 
     	this.day = this.day - subDay;
-    	while (this.day <= 0) {
+    	while (this.day < 0) {
```
Comment (get-a-clue) [inline src/Date.java:83]: Maybe <= ?

File: src/Date.java [hunk 3]
```diff
@@ -79,46 +78,36 @@ public String displayDate() {
...
     		this.month--;
+    		while(this.month <= 0) {
+    			this.year--;
+    			this.month = 12;
+    		}
     		maxNumDays = getMaxNumDays(this.month, this.year);
     		this.day = maxNumDays + this.day;
```
Comment (get-a-clue) [inline src/Date.java:90]: this.day = this.day + getMaxNumDays(this.month, this.year);

File: src/Date.java [hunk 4]
```diff
@@ -79,46 +78,36 @@ public String displayDate() {
     }
     
     public void subscDays(int subDay) {
-    	int maxNumDays = getMaxNumDays(this.month, this.year);
+    	int maxNumDays = getMaxNumDays(this.month, this.year);
```
Comment (get-a-clue) [inline src/Date.java:81]: Here should be check for positive subDay

File: src/DateTest.java [hunk 5]
```diff
@@ -92,13 +92,46 @@ void testAddDay2() {
...
 	void testSubDay() {
-		Date d = new Date(4,2,2000);
+		Date d = new Date(4,2,2001);
 		d.subscDays(5);
-		assertEquals(d.displayDate(), "The current date is 30/1/2000");
+		assertEquals(d.displayDate(), "The current date is 30/1/2001");
+		
+	}
+	
+	@Test
```
Comment (get-a-clue) [inline src/DateTest.java:112]: Please, create groups for unit tests.

File: src/Date.java (hunk 2 above)
Comment (evakaz) [reply src/Date.java:83]: thanks!

File: src/Date.java (hunk 3 above)
Comment (evakaz) [reply src/Date.java:90]: considered.

File: src/Date.java (hunk 4 above)
Comment (evakaz) [reply src/Date.java]: Thank you

File: src/DateTest.java (hunk 5 above)
Comment (evakaz) [reply src/DateTest.java:112]: will be considered.

**Pull request merged**